packages/project-access/src/project/search.ts

```diff
@@ -499,3 +499,18 @@ export async function findCapProjects(options: {
     }
     return Array.from(result);
 }
+
+/**
+ * Returns true if the specified target path contains a CAP project.
+ * Checks sub and root directories.
+ *
+ * @param targetDir the target directory path.
+ * @returns true if CAP Project is in the directory or false if not.
+ */
+export async function checkPathForCapApp(targetDir: string): Promise<string | boolean> {
```

Comment:
nit-pick: I would prefer to name this function something like `isPathForCapApp`. There's not a real check being done and you also named the property in `validators.ts` accordingly `isCapPath`. So why not the whole function?

```ts
const isCapPath = await checkPathForCapApp(targetDir);
```
looks a bit odd from my point of view

Reply:
Thank you, changed as requested.